pipelines/test/ldd-check.yaml

```diff
@@ -62,17 +62,22 @@ pipeline:
         fail "$f: missing ${missing# }"
       }
 
+      is_dyn_obj() {
+        # ldd has this code already, let's not NIH it
+        # note: the actual ldd check happens in test_file()
+        ldd "$1" > /dev/null 2>&1
```

Comment:
Does it by chance edit with a different code for "not a dynamic object" than for "catastrophic failures"

Reply:
It does not. But we can check it's error message. See the updated commit.